recipes/quast/gnuplot_issue.patch

```diff
@@ -0,0 +1,61 @@
+From ffc589098223e021f604ae033cbd2574a1de58f3 Mon Sep 17 00:00:00 2001
+From: Alexey Gurevich <alexeigurevich@gmail.com>
+Date: Mon, 25 Dec 2017 15:22:42 +0300
+Subject: [PATCH] Temp: disable gnuplot compilation and run
+
+---
+ quast_libs/ca_utils/align_contigs.py | 6 +++---
```

Comment:
In https://github.com/ablab/quast/blob/quast_4.6.1/quast_libs/contigs_analyzer.py#L260-L261
```diff
-    if qconfig.draw_plots:
-    compile_gnuplot(logger, only_clean=False)
```
should be patched too, I guess.